backend/app/utils/FfMpeg.scala

```diff
@@ -0,0 +1,24 @@
+package utils
+
+import java.nio.file.Path
+import scala.sys.process.{Process, ProcessLogger, stdout}
+
+object FfMpeg extends Logging {
+
+  case class FfMpegSubprocessCrashedException(exitCode: Int, stderr: String) extends Exception(s"Exit code: $exitCode: ${stderr}")
+
+  def convertToWav(originalFile: Path, tmpDir: Path): Path = {
```

Comment:
maybe worth adding a comment explaining that whisper.cpp only works on 16 bit wavs and linking to the docs https://github.com/ggerganov/whisper.cpp#quick-start

Reply:
Good suggestion, added the comment 👍